src/dftbp/math/eigensolver.F90

```diff
@@ -1813,7 +2062,7 @@ end subroutine dblecmplx_zhegvr
 #:for DTYPE, VPREC, VTYPE, NAME in [('real', 's', 'real', 'sgeev'), ('dble', 'd', 'real', 'dgeev')]
 
   !> Simple general matrix eigensolver
-  subroutine ${DTYPE}$_${NAME}$(a, wr, wi, vl, vr, err)
+  subroutine ${DTYPE}$_${NAME}$(a, wr, wi, vl, vr, err, info)
```

Comment:
The variables `err` and `info`  are redundant. We should eliminate `err`, so that the routine interface is similar to all other eigensolver wrapper interfaces.

Reply:
Simply replacing err by info will not work. 
Can you have a look at the routine? In case err is not present the value of err is undefined. What is 
 @:FORMATTED_ERROR_HANDLING(err, info_,
exactly doing? What is the function of the first parameter in this call?